std_msgs/Header header 
string[] nodes
builtin_interfaces/Duration[] jitter
builtin_interfaces/Duration[] latency
